libs/interpreter-lib/src/interpreter.ts

```diff
@@ -2,19 +2,19 @@
 //
 // SPDX-License-Identifier: AGPL-3.0-only
 
-// eslint-disable-next-line unicorn/prefer-node-protocol
-import { strict as assert } from 'assert';
+// eslint-disable-next-line no-restricted-imports
```

Comment:
What does that error mean?

Reply:
Thx, nice catch. Disabled the wrong linter rule here :)